locust/runners.py

```diff
@@ -148,6 +149,17 @@ def kill_locusts(self, kill_count):
             self.locusts.killone(g)
         events.hatch_complete.fire(user_count=self.num_clients)
 
+    def select_file(self, key):
+        """
+            Set the active locust classes to the executeables described by the key
+        """
+        try:
+            self.locust_classes = self.available_locustfiles[key].values()
+            print("SELECTED FILE VALUE ", self.locust_classes)
```

Comment:
remove this, or use `logger.info` instead of `print`. and if you want to log this, the I recommend to use better message. example : `Selected File : android/no-session/home_production.py`